auto_submit/lib/service/validation_service.dart

```diff
@@ -334,8 +335,13 @@ Exception: ${exception.message}
     final github.RepositorySlug slug = messagePullRequest.base!.repo!.slug();
     final int number = messagePullRequest.number!;
     // Pass an explicit commit message from the PR title otherwise the GitHub API will use the first commit message.
-    String commitMessage = messagePullRequest.title!;
-    commitMessage = commitMessage.replaceFirst('Revert "Revert', 'Reland'); // Cleanup auto-generated revert messages.
+    final String prTitle =
+        messagePullRequest.title!.replaceFirst('Revert "Revert', 'Reland'); // Cleanup auto-generated revert messages.
+    final String commitMessage = ('''
+$prTitle
+
+${messagePullRequest.body ?? ''}
```

Comment:
Should we filter out the checkboxes in the default pr template?

Reply:
considered this. rather than matching the full text (which would be brittle), what about skipping any line that starts with `- [ ]` or `- [x]`?

Reply:
That SGTM, that should remove most of the noise